exercises/secret-handshake/instructions.md

```diff
@@ -2,7 +2,7 @@
 
 Your task is to convert a number between 1 and 31 to a sequence of actions in the secret handshake.
 
-The sequence of actions is chosen by looking at the rightmost five digits of the number once it's been converted to binary.
+The sequence of actions is chosen by looking at the five digits of the number (which is between 1 and 31, equals 2 ^ 5 - 1) once it's been converted to binary. 
```

Comment:
We had `rightmost` because a lot of people think of a binary number as being 8 bits. So converting 17 to binary would give you: `00010001`.

I don't want to add an equation in the main text. We already have a note saying to go look up how binary works if you don't know. If we think we need to explain more, then we need to explain in the note, not in the main body of the instructions.

We don't need to repeat the fact that the number is between 1 and 31.

Reply:
OK, I'll revert this, then.